### DISCUSSION TITLE
Records Keeping Statement

### DISCUSSION THREAD

**Discussion by clementbiron** [OpenTermsArchive/terms-types#78] [Qualified]
### What are some documents containing these terms?

| Example | Service Name | URL | Title | Jurisdiction(s) |
|:----|:----|:----|:------|:-------------|
| 1 | Cuntempire | [URL 🔗](https://www.cuntempire.com/docs/2257.html) | 18 U.S.C. 2257 Record-Keeping Requirements Exemption Statement | 🇺🇸 |
| 2 | Pornpics | [URL 🔗](https://www.pornpics.com/2257/) | 18 U.S.C. § 2257 Statement | 🇺🇸 |
| 3 | RabbitsReviews | [URL 🔗](https://www.rabbitsreviews.com/legal#USC18) | 18 U.S.C. § 2257 Statement | 🇺🇸 |
| 4 | RedGIFs | [URL 🔗](https://www.redgifs.com/notice2257) | 18 U.S.C 2257 Statement | 🇺🇸 |
| 5 | Pokmi | [URL 🔗](https://legal.pokmi.com/18-u-s-c-%C2%A72257-record-keeping-requirements-compliance-statement/) | 18 U.S.C. §2257 Record Keeping Requirements Compliance Statement | 🇺🇸 |

### What could be some alternative names for this type?

18 U.S.C 2257 Statement
18 USCC 2257 Statement
2257 Statement
Records Keeping Policy
Records Keeping Requirements
Record-Keeping Requirements Exemption Statement
Record Keeping Requirements Compliance Statement

### Whose rights and duties are defined in these terms?

end user

### What is the object of commitment in these terms?

requirements on producers of visual depictions of actual human beings engaged in actual or simulated sexually explicit conduct

### Are there any relevant references for this type?

- [Wikipedia page on Child Protection and Obscenity Enforcement Act](https://en.wikipedia.org/wiki/Child_Protection_and_Obscenity_Enforcement_Act)
- [18 U.S. Code § 2257](https://www.law.cornell.edu/uscode/text/18/2257)

**Comment by Ndpnt** [comment]
One example is missing to move this discussion as `Qualified`

**Reply by clementbiron** [reply]
I've added one :+1:

**Comment by Ndpnt** [comment]
Does this terms type only available in US jurisdiction? Could you find examples in another jurisdiction?

**Reply by MattiSG** [reply]
Yes, this seems to be mandated by a US legal act.

**Comment by MattiSG** [comment]
- Considering this is a compliance statement, I would recommend to rename into “Records Keeping Statement”, with alternative names being “18 U.S.C 2257 Statement”, “18 USCC 2257 Statement”, “Record-Keeping Requirements Exemption Statement”, “Record Keeping Requirements Compliance Statement”.
- I'm adding a legal reference on top of Wikipedia.
